python.d/README.md

```diff
@@ -1742,10 +1742,63 @@ localhost:
   pass : 'secret_tomcat_password'
 ```
 
-Without configuration, module attempts to connect to `http://localhost:8080/manager/status?XML=true`, without any credentials. 
+Without configuration, module attempts to connect to `http://localhost:8080/manager/status?XML=true`, without any credentials.
 So it will probably fail.
 
---- 
+---
+
+# Traefik
+
+Module uses the `health` API to provide statistics.
+
+It produces:
+
+1. **Uptime**
+ * Traefik server uptime
+
+2. **Average Response Time**
+ * average response time
+
+3. **Total Response Time**
+ * sum of all response time
+
+4. **Requests** requests/s
+ * request statistics
+
+5. **Response by type** requests/s
+  * success (1xx, 2xx, 304)
+  * error (5xx)
+  * redirect (3xx except 304)
+  * bad (4xx)
+  * other (all other responses)
+
+6. **Response by code family** requests/s
+  * 1xx (informational)
+  * 2xx (successful)
+  * 3xx (redirect)
+  * 4xx (bad)
+  * 5xx (internal server errors)
+  * other (non-standart responses)
+  * unmatched (the lines in the log file that are not matched)
```

Comment:
6 charts but [your dashboard](https://monitoring.bsecteam.com/#menu_traefik_local_submenu_responses;theme=slate;help=true) has 8
And please fix the order here too

Reply:
`README.md` fixed.